### PR TITLE
User documentation: Meals section of Admin-UI

### PR DESCRIPTION
**Pull request by invisible-pvi**
**Describe the technical changes contained in this PR**
Created a documentation for the Meal section of the Admin-UI page of the MealPlanner app. 

**Previous behaviour**
Previously, documentation for the Meal section of the Admin-UI was not present. So, I created a new document page allowing users to view the Admin-UI Meal section documentation. 

**New behaviour**
The documentation tool used is Mkdocs. So, now the documentation for the MealPlanner app would have a new section that describes that wide-variety of functionalities for the Meals section of the Admin-UI part of the app. 

**Related issues addressed by this PR**

**Have the following been addressed?**
- [ ] Have test cases been created for all of the changes?
- [ ] Do all of the test cases pass?
- [ ] Has the testing been done using the default docker-compose environment?
- [ ] Are documentation changes required?
- [ ] Does this change break or alter existing behaviour?

### CODE REVIEW

File: docs/admin-ui-meals.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+2. **Select the Meal tab.**
+    - Click the Meal tab on the top left sidebar of the admin-ui page of the MealPlanner app, similar to the screenshot below. 
+
+
+
+3. **Click the Create button**
+    - Click the `Create` button in the top right part of the admin-ui page. 
+    
+<img width="800" alt="image" src="create-meal.jpeg">
```
Comment (rorybray) [inline docs/admin-ui-meals.md]: Markdown image markup should look like this: Create Meal

HTML style img tags won't work properly in the generated site.

**Review by rorybray — CHANGES_REQUESTED** [review]
Really good documentation, just need to fix the images.

File: docs/admin-ui-meals.md (hunk 1 above)
Comment (invisible-pvi) [reply docs/admin-ui-meals.md]: Thank you! I have made the changes to fix the image addition into the md file.

**Comment by rorybray** [comment]
Looks good to me

**Review by rorybray — APPROVED** [review]

**Pull request merged**